src/core/resources/assets/userTestnetNativeAsset.ts

```diff
@@ -19,11 +19,15 @@ const USER_ASSETS_REFETCH_INTERVAL = 60000;
 
 export const getNativeAssetMock = ({ chainId }: { chainId: ChainId }) => {
   const chain = getChain({ chainId });
+  const { address: nativeAssetAddress } = networkStore
+    .getState()
+    .getChainsNativeAsset()[chainId];
+  const chainLabel = networkStore.getState().getChainsLabel()[chainId];
   const nativeAssetMock = {
-    address: chainsNativeAsset[chainId],
+    address: nativeAssetAddress as AddressOrEth,
     balance: { amount: '', display: '' },
     chainId: chainId,
-    chainName: chainsLabel[chainId] as ChainName,
+    chainName: chainLabel as ChainName,
```

Comment:
this looks like an existing issue, but `chainName` should be from `getChainsName` instead of `getChainsLabel` to be consistent